keps/168-2-pending-workloads-visibility/README.md

```diff
@@ -209,7 +209,7 @@ type PendingWorkload struct {
 
 // PendingWorkloadSummaryList contains a list of pending workloads in the context
 // of the query (within LocalQueue or ClusterQueue).
-type PendingWorkloadsSummaryList struct {
+type PendingWorkloadsSummary struct {
```

Comment:
I think it's a better idea to change it into `PendingWorkloadList` so it's similar to what we already have in Kueue:
https://github.com/kubernetes-sigs/kueue/blob/cf25c71bef51fa6d0880432352dd92da6f3a8ea6/apis/kueue/v1beta1/workload_types.go#L293

Reply:
I'm fine with `PendingWorkloadsSummary` and `PendingWorkloadList`, slightly leaning towards `PendingWorkloadList` as the `List` sufix seems like a convention.

Reply:
It is a convention for list endpoints. This is not a list endpoint.

Reply:
Makes sense, let's go with `PendingWorkloadsSummary` for the Get endpoint.

Reply:
Thanks for the clarification.